dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -26,24 +28,24 @@
 from dbt.events import AdapterLogger
 from dbt.utils import DECIMALS
 from requests_toolbelt.multipart.encoder import MultipartEncoder
-from types import TracebackType
 from typing import Any
-from urllib.parse import urlencode
 
+# fmt: on
 logger = AdapterLogger("Spark")
 adapter_timer = AdapterTimer()
 
-DEFAULT_POLL_WAIT = 2  # seconds
-DEFAULT_LOG_WAIT = 10  # seconds
-DEFAULT_RETRIES = 10  # max number of retries for fetching log
-MIN_LINES_TO_PARSE = (
-    3  # minimum lines in the logs file before we can start to parse the sql output
+DEFAULT_POLL_WAIT = 30  # time to sleep in seconds before re-fetching job status
```

Comment:
This is still a magic number. But i found 30 seconds to be a happy medium for polling job status. Too quick polling leads to throttling and connection timeouts. Too slow polling adds to latency.